buildscript.gradle

```diff
@@ -15,38 +15,35 @@
  */
 
 rootProject.buildscript {
-  String rosMavenPath = System.getenv("ROS_MAVEN_PATH")
-  String rosMavenRepository = System.getenv("ROS_MAVEN_REPOSITORY")
-
-  repositories {
-    if (rosMavenPath != null) {
-      rosMavenPath.tokenize(":").each { path ->
+    String rosMavenPath = System.getenv("ROS_MAVEN_PATH")
+    String rosMavenRepository = System.getenv("ROS_MAVEN_REPOSITORY")
+    repositories {
+        if (rosMavenPath != null) {
+            rosMavenPath.tokenize(":").each { path ->
+                maven {
+                    // We can't use uri() here because we aren't running inside something
+                    // that implements the Script interface.
+                    url "file:${path}"
+                }
+            }
+        }
         maven {
-          // We can't use uri() here because we aren't running inside something
-          // that implements the Script interface.
-          url "file:${path}"
+            url "http://repository.springsource.com/maven/bundles/release"
         }
-      }
-    }
-    maven {
-      url "http://repository.springsource.com/maven/bundles/release"
-    }
-    maven {
-      url "http://repository.springsource.com/maven/bundles/external"
+        maven {
+            url "http://repository.springsource.com/maven/bundles/external"
+        }
+        if (rosMavenRepository != null) {
+            maven {
+                url rosMavenRepository
+            }
+        }
+        maven {
+            url "https://github.com/rosjava/rosjava_mvn_repo/raw/master"
+        }
+        mavenCentral()
     }
-    if (rosMavenRepository != null) {
-      maven {
-        url rosMavenRepository
-      }
-    } else {
-      maven {
-        url "https://github.com/rosjava/rosjava_mvn_repo/raw/master"
-      }
+    dependencies {
+        classpath "org.ros.rosjava_bootstrap:gradle_plugins:[0.2,0.3)"
     }
-    mavenCentral()
-  }
-
-  dependencies {
-    classpath "org.ros.rosjava_bootstrap:gradle_plugins:[0.2,0.3)"
-  }
 }
```

Comment:
i want to add to condition || '' != rosMavenRepository
mean if it will be empty that it will not add it as a maven.
maybe at windows it take it as empty. i am not sure.
but also it doesn't , it make our plugin more safer.